content/projects/jcasc/roadmap/data.json

```diff
@@ -0,0 +1,80 @@
+{
+  "statuses": [
+    "not_planned",
+    "planned",
+    "in_progress",
+    "ready_soon",
+    "released"
+  ],
+  "categories": {
```

Comment:
Maybe it's easier to just track Milestones in GitHub and add links. Though it allows creating a well-looking page

Reply:
I used Blue Ocean as a template because I really like how it looks, but I'm worrying about updating it every time we plan new release or change the scope. So maybe tracking Milestones is not a bad idea - do you have an example of such page?